src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,65 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+
+/**
+ * Handles fetching and saving user data.
+ */
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet {
+
+  private Datastore datastore;
+
+ @Override
+ public void init() {
+  datastore = new Datastore();
+ }
+ 
+ /**
+  * Responds with the "about me" section for a particular user.
+  */
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+   throws IOException {
+
+  response.setContentType("text/html");
+  
+  String user = request.getParameter("user");
+  
+  if(user == null || user.equals("")) {
+   // Request is invalid, return empty response
+   return;
+  }
+  
+  String aboutMe = "This is " + user + "'s about me.";
```

Comment:
@jimenezjose - I would look at using `String.format` here - see https://dzone.com/articles/java-string-format-examples

e.g. `String aboutMe = String.format("This is %s's About Me", user);`